charts/camunda-platform-8.3/values.yaml

```diff
@@ -1222,7 +1222,7 @@ optimize:
   migration:
     ## @param optimize.migration.enabled if true, run Optimize migration script as an init container
     enabled: true
-    ## @param optimize.migration.env can be used to set environment variables for Optimize migration init container
+    ## @param optimize.migration.env can be used to set environment variables for Optimize migration init container (Note: there is no need to define if optimize.env is defined)
```

Comment:
I notice you update values.yaml in v8.3 of the helm charts, but you do not update the optimize deployment.yaml template.

Reply:
I fixed this myself. b150208fb